### ISSUE TITLE
[amp-story-player] Ability to make the play/pause button visible in non desktop sizes

### ISSUE THREAD

**Issue by pshihn** [ampproject/amphtml#31332]
## Describe the new feature or change to an existing feature you'd like to see

Reading at the documentation of the amp-story-player, it seems like the only way to show the play/pause button in an embedded player is to build your own control. You cannot make the built-in play/pause button to appear. 

This seems like extra work for no reason. Even if you build your own, it will appear either at the start of the controls or at the end, which does not feel natural. The built in play/pause usually appears before the share button. 

I do realize it shows up when the aspect ration is *desktop* but when using a player to embed on a web page you may choose a mobile like aspect ratio. 

## Describe alternatives you've considered

I can write my own custom button to control the play state by listening to events, but it seems like redundant work

**Comment by pshihn** [comment]
Actually it seems like there are no events for pause/play either so cannot easily create a custom button either.

**Comment by pshihn** [comment]
Also **there is no way to pause** a video using the amp-story-player by default if you are not on a mobile touch device. I tried holding down the mouse button to see if it did it, but nope.

**Comment by gmajoulet** [comment]
cc @ampproject/wg-stories

**Comment by newmuis** [comment]
@Enriqe can you change the order of the icons through the custom controls API?

**Comment by Enriqe** [comment]
Nope, currently the order is defined by the system layer of the story format @newmuis

**Comment by arbazkhalid1122** [comment]
we can also use this approach for play and pause like onClick or however you want like in react js 


  useEffect(() => {
    const pressKey = (event) => {
      if (event.code === 'Space' || event.code === ' ') {
        event.preventDefault();
        if (ampStoryRef.current) {
          const element = ampStoryRef.current;
          const systemShadowParent = element.querySelector(".i-amphtml-system-layer-host");
          const root = systemShadowParent?.shadowRoot;
          const pause = root?.querySelector(".i-amphtml-story-pause-control");
          const play = root?.querySelector(".i-amphtml-story-play-control");        
            if (pause && !isPaused) {
              pause.click();
            } else if (play && isPaused) {
              play.click();
            }
        }
      }
    };
    window.addEventListener('keydown', pressKey);
    return () => {
      window.removeEventListener('keydown', pressKey);
    };
  }, [ampStoryRef, isPaused, showForm]);
